### PR TITLE
[Dashboard] Add batch claim feature <dont auto merge>

### PR DESCRIPTION
**Pull request by kien-ngo**
---

## PR-Codex overview
This PR introduces enhancements to the airdrop functionality across various components, allowing for optional removal of duplicate addresses and improving the user interface for claiming NFTs.

### Detailed summary
- Added `removeDuplicateAddresses` prop to `AirdropUploadProps`.
- Implemented logic to filter out duplicate addresses in `processAirdropData`.
- Updated `AirdropUpload` to handle duplicate addresses.
- Introduced `SingleClaimTab` and `BatchClaimTab` for NFT claiming.
- Improved UI with tabbed navigation for single and batch claims.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5154?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5154** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5154?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#5127** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5127?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by kien-ngo** [comment]
DASH-343

**Review by MananTank — DISMISSED** [review]
please address comments

File: apps/dashboard/src/@/components/ui/tabs.tsx [hunk 1]
```diff
@@ -8,6 +11,55 @@ import { useCallback, useRef, useState } from "react";
 import { ScrollShadow } from "./ScrollShadow/ScrollShadow";
 import { Button } from "./button";
 
+const Tabs = TabsPrimitive.Root;
```
[the remark this answers is not in the document]
Comment (kien-ngo) [reply apps/dashboard/src/@/components/ui/tabs.tsx]: done

**Review by MananTank — COMMENTED** [review]
Deferring the review to @jnsdls for this PR

**Pull request closed without merging**